full-test

```diff
@@ -0,0 +1 @@
+TODO delete me
```

Comment:
In case you missed it, I made sure that the presence of this file will result in a build failure _after_ running all tests, so you cannot accidentally automerge. :grin: 

Currently this would also block incremental deployment of the BOM. I think that could be fixed in case anyone uses incremental BOM versions, though I do not recall seeing anyone do so recently, since it is usually easy enough to just leave in a temporary `dependencyManagement` override. (Not to be confused with the _BOM_ build using an incremental version of a _plugin_.)